lib/changelog/oban_workers/slack_importer.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Changelog.ObanWorkers.SlackImporter do
+  use Oban.Worker, queue: :scheduled
+
+  alias Changelog.{Person, Repo}
+  alias Changelog.Slack.Client
+
+  import Ecto.Query
+
+  @impl Oban.Worker
+  def perform(_job) do
+    %{"members" => members} = Client.list()
+
+    for %{"id" => id, "profile" => profile} <- members do
+      email = Map.get(profile, "email", "")
+
+      import_member_id(id, email)
+    end
+
+    :ok
+  end
+
+  def import_member_id(id, email) do
+    Person
+    |> where([p], p.email == ^email)
+    |> where([p], is_nil(p.slack_id) or p.slack_id == "pending")
+    |> Repo.update_all(set: [slack_id: id])
```

Comment:
The use of `update_all` here allows for a single DB call, rather than a read/write cycle. It's pretty minor considering that I have no idea how many users are in the slack channel, but it was low fruit 🙂